usr.bin/sockstat/sockstat.c

```diff
@@ -1116,6 +1126,10 @@ format_unix_faddr(struct addr *faddr, char *buf, size_t bufsize) {
 			if (f != NULL) {
 				pos += snprintf(SAFEBUF, SAFESIZE, "[%lu %d]",
 					(u_long)f->xf_pid, f->xf_fd);
+				if (!is_text_style) {
+				    xo_emit("{:pid/%lu}", (u_long)f->xf_pid);
+				    xo_emit("{:fd/%d}", f->xf_fd);
```

Comment:
Style: spaces to tabs

Reply:
Also, for consistency's sake you should add a `xo_open_list` here, even though it will always have only one element.  That will make it easier for consumers to parse .